docs/guide/apm-release-notes.asciidoc

```diff
@@ -25,6 +26,17 @@ For a full list of changes, see the
 
 // end::notable-v8-highlights[]
 
+[[release-highlights-7.5.0]]
```

Comment:
here or somewhere else: agent config goes GA, with new settings, RUM, support for "All services" (in Kibana).
GA breaks with beta, so a 7.5 apm-server will require 7.5 kibana and viceversa.